### PR TITLE
Add optional links to be displayed in the third column with a top border 

### PR DESCRIPTION
**Pull request by chillkang**
## Done
Add optional links to be displayed in the third column with a top border 

## QA
-Go to https://store-components-22.demos.haus/?path=/story/footer--default-story
-Check if there are optional links in the third column with a top border

## Fixes
Fixes: https://warthogs.atlassian.net/browse/WD-1616

### CODE REVIEW

File: src/components/Footer/Footer.tsx [hunk 1]
```diff
@@ -56,6 +62,24 @@ function Footer({ socialLinks, additionalLinks }: Props) {
               ))}
           </ul>
         </Col>
+        <Col size={6}>
+          <p className="u-align--left">
+            Ubuntu and Canonical are registered trademarks of Canonical Ltd.
+            <br></br>
+            Powered by{" "}
+            <a href="https://www.ubuntu.com/kubernetes">Charmed Kubernetes</a>
```
Comment (steverydz) [inline src/components/Footer/Footer.tsx:70]: I think we can remove this line

File: src/components/Footer/Footer.tsx [hunk 2]
```diff
@@ -56,6 +62,24 @@ function Footer({ socialLinks, additionalLinks }: Props) {
...
+          </p>
+          <hr></hr>
```
Comment (steverydz) [inline src/components/Footer/Footer.tsx:72]: This `hr` should only show if there are optional links. Also please use `<hr />`

File: src/components/Footer/Footer.tsx [hunk 3]
```diff
@@ -56,6 +62,24 @@ function Footer({ socialLinks, additionalLinks }: Props) {
...
+          {optionalLinks &&
+            optionalLinks.map((optionalLink) => (
+              <li className="p-inline-list__item" key={optionalLink?.name}>
+                <a href={optionalLink?.href}>
+                  {optionalLink?.name}
+                  <i className="p-icon--external-link"></i>
```
Comment (steverydz) [inline src/components/Footer/Footer.tsx:78]: We don't need to use this icon

**Review by steverydz — CHANGES_REQUESTED** [review]

File: src/components/Footer/Footer.tsx (hunk 1 above)
Comment (chillkang) [reply src/components/Footer/Footer.tsx]: I removed it :)

File: src/components/Footer/Footer.tsx (hunk 3 above)
Comment (chillkang) [reply src/components/Footer/Footer.tsx]: I removed this also.

**Review by steverydz — APPROVED** [review]
LGTM 👍

**Pull request merged**